layouts/default.vue

```diff
@@ -1,43 +1,62 @@
+const darkMode = ref( false )
+const atTop = ref( true )
 
-onMounted(() => {
-  document.addEventListener('scroll', (e) => {
+onMounted( () => {
+  document.addEventListener( 'scroll', ( e ) => {
     atTop.value = window.scrollY > 0 ? false : true
     //atBottom.value = ((window.scrollY + (window.innerHeight + 115) >= document.body.scrollHeight)) ? true : false
-  })
-})
+  } )
+  // Ads
+  window.htlbid = window.htlbid || {};
+  htlbid.cmd = htlbid.cmd || [];
+  htlbid.cmd.push( function () {
+    htlbid.layout( 'universal' ); // Leave as 'universal' or add custom layout
+    htlbid.setTargeting( "is_testing", "yes" ); // Set to "no" for production
+    htlbid.setTargeting( "is_home", "no" ); // Set to "yes" on the homepage
+    htlbid.setTargeting( "category", "INSERT_CATEGORY_HERE" ); // dynamically pass page category into this function
+    htlbid.setTargeting( "post_id", "INSERT_POSTID_HERE" ); // dynamically pass unique post/page id into this function
+  } );
+} )
 </script>
 
 <template>
-  <div class="page" :class="[`${route.name}`]" :data-style-mode="darkMode ? 'dark' : 'default'">
+  <div class="page" :class=" [ `${ route.name }` ] " :data-style-mode=" darkMode ? 'dark' : 'default' ">
     <Html>
-      <Head>
-        <Title>Radiolab: Podcasts | WNYC Studios | Podcasts</Title>
-        <Meta name="description" content="Investigating a strange world." />
-        <Meta name="keywords" content="podcasts, npr, new york, WNYC Studios, arts, culture, classical, music, news, public, radio" />
-        <Meta name="og:site_name" content="Radiolab Podcasts | WNYC Studios" />
-        <Meta name="og:type" content="website" />
-        <Meta name="og:url" :content="`https://www.radiolab.org${route.fullPath}`" />
-        <Meta name="og:title" content="Radiolab: Podcasts | WNYC Studios | Podcasts" />
-        <Meta name="og:description" content="Investigating a strange world." />
-        <Meta name="og:image" content="https://media.wnyc.org/i/1200/600/l/80/2020/09/Radiolab_OG_Image_Options_3.png" />
-        <Meta name="og:image:width" content="1200" />
-        <Meta name="og:image:height" content="600" />
-        <Meta name="fb:app_id" content="151261804904925" />
-        <Meta name="fb:pages" content="224921207956535" data-detail="WNYC Studios" />
-        <Meta name="twitter:card" content="summary_large_image" />
-        <Meta name="twitter:site" content="@radiolab" />
-        <Meta name="twitter:title" content="Radiolab: Podcasts | WNYC Studios | Podcasts" />
-        <Meta name="twitter:description" content="Investigating a strange world." />
-        <Meta name="twitter:image" content="https://media.wnyc.org/i/1200/600/l/80/2020/09/Radiolab_OG_Image_Options_3.png" />
-        <Meta name="apple-itunes-app" content="app-id=152249110, affiliate-data=at=1010lSme&amp;ct=radiolab" />
-      </Head>
+
+    <Head>
+      <Link v-if=" config.ENV === 'prod' " rel="stylesheet" href="https://htlbid.com/v3/radiolab.com/htlbid.css"
```

Comment:
a better way to do this would be to set the stylesheet ref in an environment variable that would be injected via circle. I'll go ahead and enter these values for the relevant environments and you can replace this with environment variables similar to how I did on my branch. That way the code itself is ignorant of the environment =)